releases/net471/KnownIssues/479874-WPF Touch Stops Working After Prolonged Use of Popups.md

```diff
@@ -7,6 +7,4 @@ On a touch or stylus-enabled machine, WPF applications may see a loss of touch o
 A change to disposal of HwndSource in .NET 4.7 caused Popups to not correctly cleanup touch resources.  This eventually can lead to loss of touch/stylus interaction.
 
 ### Resolution
-The fix for this is expected in future servicing update for Windows 10.
-
-### More information
\ No newline at end of file
+This issue is fixed for all supported OS platforms prior to Windows 10 Fall Creators Update. The fix for Windows 10 Fall Creators Update is expected in future servicing update. 
```

Comment:
future servicing update --> a future servicing update

The change should be made in each of the following files in which this phrase is found.

Reply:
Will fix in all places